Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-archiving.controller.ts

```diff
@@ -41,17 +41,19 @@
             $scope.systemUsage = systemUsage;
             $scope.archivePeriods = archivePeriod;
             $scope.hasWriteAccessAndArchived = systemUsage.Archived;
-            $scope.ArchiveDuty = systemUsage.ArchiveDuty;
+            $scope.ArchiveDuty = $scope.usage.archiveDuty;
             $scope.archiveReadMoreLink = Kitos.Constants.Archiving.ReadMoreUri;
             $scope.translateArchiveDutyRecommendation = (value: number) => Kitos.Models.ItSystem.ArchiveDutyRecommendationFactory.mapFromNumeric(value).text;
             $scope.archiveDutyOptions = Kitos.Models.ItSystemUsage.ArchiveDutyOptions.getAll();
 
+            $scope.autoSaveUrl = 'api/itSystemUsage/' + $stateParams.id;
+
             if (!systemUsage.Archived) {
                 $scope.systemUsage.Archived = false;
```

Comment:
I tried removing the "systemUsage" (from what I can see it does the same thing as the "itSystemUsage", but every time you enter the tab), but I couldn't find how to get the "Archived" property

Reply:
I think a lot of the other properties bind to this odata thing...
Ideally we should remove the duplicate binding and only use one or the other. For this page though, we will wait until the UI revamp where we will change to use the V2 api